### PR TITLE
Added documentation for the changeTitle

### PR DESCRIPTION
**Pull request by RachBreeze**
Hello,

Thank you for responding to my question here https://github.com/umbraco/UmbracoDocs/issues/2770 
I've added missing documentation for the changeTitle directive.  This is my first PR to the docs, I am really keen to get feedback to ensure I've followed the preferred structure and documented the directive correctly.

Thank you in advance for your time

(Created on CanConHack Day)

### CODE REVIEW

File: Reference/Angular/Directives/changeTitle/Index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+versionFrom: 8.4.0
+---
+
+
+# changeTitle
+
+Setting the title of the page the user is working is important accessibility purposes; where people using assitive technology need to know what they are maintaining.
```
Comment (marcemarc) [inline Reference/Angular/Directives/changeTitle/Index.md:8]:
```suggestion
Setting the title of the page the user is working on is important for accessibility purposes; where people using assistive technology need to know what they are maintaining. 
```

File: Reference/Angular/Directives/changeTitle/Index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+versionFrom: 8.4.0
+---
+
+
+# changeTitle
+
+Setting the title of the page the user is working is important accessibility purposes; where people using assitive technology need to know what they are maintaining. 
+By setting the page title people who work with multiple tabs will also find the page they were working on easily.
```
Comment (marcemarc) [inline Reference/Angular/Directives/changeTitle/Index.md:9]:
```suggestion
By setting the page title, people who work with multiple tabs will also find the page they were working on easily.
```

**Review by marcemarc — COMMENTED** [review]
Hi @RachBreeze 

This is a great explanation of the directive and how it can be used for assistive purposes.

I spotted a typo, and made a suggestion, the only other thought is whether it would be worth having a screenshot of where the title is updated?

**Comment by RachBreeze** [comment]
Hi @marcemarc thank you, and the screen is  a great idea leave it with me hopefully I can something back to you in the next couple of days

**Comment by sofietoft** [comment]
Thanks for updating the article @RachBreeze !
I think it looks great - I'll merge it in!

Thanks again for putting in the effort to write this article - much appreciated! 😄

**Comment by marcemarc** [comment]
> Hi @marcemarc thank you, and the screen is a great idea leave it with me hopefully I can something back to you in the next couple of days

Thanks @RachBreeze !

**Comment by abjerner** [comment]
Hi everyone. I just spotted this, and learned something new, and the content of the page seems spot on 👍 

I do however have some concerns about the location of the file, as this is not really a directive. Setting the title utilizes the `eventsService`, so perhaps should be part of the page below instead?

https://our.umbraco.com/documentation/Reference/Angular/Services/eventsService/

Or have it's own page in the structure under the page about `eventsService` to make it more visible? Or somewhere else?

What do you think?

**Comment by RachBreeze** [comment]
Heya
Thank you @abjerner 
I am still learning the docs, so I'm happy to move to the `eventsService`

**Comment by sofietoft** [comment]
Ah! Good catch @abjerner 💪 Makes sense to move it under `eventService`!

And you're up for making the move @RachBreeze ? 😄

**Comment by RachBreeze** [comment]
@sofietoft yes 100%

**Comment by abjerner** [comment]
It may make sense to add/move it as a page below `eventsService`, and then add a link to from the `eventsService` page. That should hopefully make it easier to find 😉

**Pull request merged**